variables.md

```diff
@@ -110,6 +110,7 @@ LTP_KNOWN_ISSUES | string | | Used to specify a url for a json file with well kn
 LTP_REPO | string | | The repo which will be added and is used to install LTP package.
 LTP_RUN_NG_BRANCH | string | | Define the branch of the LTP_RUN_NG_REPO.
 LTP_RUN_NG_REPO | string | | Define the runltp-ng repo to be used. Default in publiccloud/run_ltp.pm is the upstream master branch from https://github.com/metan-ucw/runltp-ng.git.
+LTP_RUNTIME | string | | Values: **python** or **perl**, default empty = perl. <p>This parameter is used to run proof and consolidation tests for the new python version of the LTP runltp-ng repository, allowing to switch between the original Perl and the new Python repo versions.</p><p>NOTE: when **python** value set, also needed to set: *LTP_RUN_NG_REPO='https://github.com/acerv/runltp-ng.git'* and *LTP_RUN_NG_BRANCH='ssh'*.</p>
```

Comment:
Line 28 in `run_ltp.pm` suggests that the default value is not empty but `perl`. Also - please don't use html tags in markdown.